bookwyrm/templates/snippets/book_cover.html

```diff
@@ -2,41 +2,136 @@
+                        type="image/jpg"
+                        srcset="{{ book.cover_bw_book_xlarge_jpg.url }}"
+                    />
+                {% elif size_mobile == 'xxlarge' %}
+                    <source
+                        media="(max-width: 768px)"
+                        type="image/webp"
+                        srcset="{{ book.cover_bw_book_xxlarge_webp.url }}"
+                    />
+                    <source
+                        media="(max-width: 768px)"
+                        type="image/jpg"
+                        srcset="{{ book.cover_bw_book_xxlarge_jpg.url }}"
+                    />
+                {% endif %}
 
-        {% if book.cover %}
-            src="{% if img_path is None %}{% get_media_prefix %}{% else %}{{ img_path }}{% endif %}{{ book.cover }}"
-            itemprop="thumbnailUrl"
+                {% if size == 'xsmall' %}
+                    <source type="image/webp" srcset="{{ book.cover_bw_book_xsmall_webp.url }}"/>
+                    <source type="image/jpg" srcset="{{ book.cover_bw_book_xsmall_jpg.url }}"/>
+                {% elif size == 'small' %}
+                    <source type="image/webp" srcset="{{ book.cover_bw_book_small_webp.url }}"/>
+                    <source type="image/jpg" srcset="{{ book.cover_bw_book_small_jpg.url }}"/>
+                {% elif size == 'medium' %}
+                    <source type="image/webp" srcset="{{ book.cover_bw_book_medium_webp.url }}"/>
+                    <source type="image/jpg" srcset="{{ book.cover_bw_book_medium_jpg.url }}"/>
+                {% elif size == 'large' %}
+                    <source type="image/webp" srcset="{{ book.cover_bw_book_large_webp.url }}"/>
+                    <source type="image/jpg" srcset="{{ book.cover_bw_book_large_jpg.url }}"/>
+                {% elif size == 'xlarge' %}
+                    <source type="image/webp" srcset="{{ book.cover_bw_book_xlarge_webp.url }}"/>
+                    <source type="image/jpg" srcset="{{ book.cover_bw_book_xlarge_jpg.url }}"/>
+                {% elif size == 'xxlarge' %}
+                    <source type="image/webp" srcset="{{ book.cover_bw_book_xxlarge_webp.url }}"/>
+                    <source type="image/jpg" srcset="{{ book.cover_bw_book_xxlarge_jpg.url }}"/>
+                {% endif %}
 
-            {% if book.alt_text %}
-                alt="{{ book.alt_text }}"
             {% endif %}
-        {% else %}
-            src="{% static "images/no_cover.jpg" %}"
-            alt="{% trans "No cover" %}"
+
+            <img
+                class="book-cover"
+                itemprop="thumbnailUrl"
+                {% if book.cover %}
+                src="{% if img_path is None %}{% get_media_prefix %}{% else %}{{ img_path }}{% endif %}{{ book.cover }}"
+                {% else %}
+                src="{% static "images/no_cover.jpg" %}"
+                alt="{% trans "No cover" %}"
+                {% endif %}
```

Comment:
I just saw that there's already a condition on the exitence of `book.cover` (L.7), so I think we can simplify this to:

```suggestion
                src="{% if img_path is None %}{% get_media_prefix %}{% else %}{{ img_path }}{% endif %}{{ book.cover }}"
```

The case for no `book.cover` is below, L.125

Reply:
This looks like something I janked up in the merge conflict. Thanks!